backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/ObservationHandler.java

```diff
@@ -87,18 +89,20 @@ public ObservationHandler(Executor executor, FutureJdbi jdbi) {
             handle ->
                 handle
                     .createQuery(
-                        "select k.kv_key _key, k.kv_value _value, k.value_type _type, o.epoch_number epoch, o.experiment_run_id run_id "
+                        "select k.kv_key _key, k.kv_value _value, k.value_type _type, o.epoch_number epoch, o.experiment_run_id run_id, o.timestamp "
                             + " from observation as o"
                             + " join keyvalue as k"
                             + " on o.keyvaluemapping_id = k.id"
-                            + " where o.experiment_run_id in (<run_ids>) and o.entity_name = \"ExperimentRunEntity\" and k.entity_name IS NULL")
+                            + " where o.experiment_run_id in (<run_ids>) and o.entity_name = :entityName and k.entity_name IS NULL")
                     .bindList("run_ids", runIds)
+                    .bind("entityName", "ExperimentRunEntity")
```

Comment:
Use extra quotes for SQLserver compatibility

Reply:
Missed to fix by Cory so fix it here